src/ProjectNeon/Assets/Features/Battle/Variables/SpellHitsLastCardMultiplier.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpellHitsLastCardMultiplier : FloatVariable
+{
+    [SerializeField] CardResolutionZone resolutionZone;
+
+    public new float Value => resolutionZone.SpellHitsOnLastCard * 0.1f;
```

Comment:
This doesn't work. No user of the FloatVariable will access this value.

Reply:
@SilasReinagel Why not? It is accesed the same way `FloatVariable`